templates/settingsForm.tpl

```diff
@@ -9,8 +9,16 @@
  *}
 <script>
 	$(function() {ldelim}
-		// Attach the form handler.
-		$('#betterPasswordSettingsForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
```

Comment:
Is the delete of this `pkpHandler` intentional?

Reply:
The form functions as expected without this Ajax Handler. The only handler required is the file upload handler.

Reply:
Can you review and share the function of the AjaxFormHander and document why it is not needed here?

Reply:
https://github.com/pkp/pkp-lib/blob/master/js/controllers/form/FileUploadFormHandler.js#L19 
The file upload handler extends the Ajax Form Handler